pkg/model/iam/tests/iam_builder_node_strict_ecr.json

```diff
@@ -7,8 +7,6 @@
       "Effect": "Allow",
       "Resource": [
         "arn:aws-test:s3:::kops-tests/iam-builder-test.k8s.local/addons/*",
-        "arn:aws-test:s3:::kops-tests/iam-builder-test.k8s.local/cluster-completed.spec",
-        "arn:aws-test:s3:::kops-tests/iam-builder-test.k8s.local/igconfig/node/*",
         "arn:aws-test:s3:::kops-tests/iam-builder-test.k8s.local/pki/private/kube-proxy/*",
         "arn:aws-test:s3:::kops-tests/iam-builder-test.k8s.local/pki/private/kubelet/*",
```

Comment:
Looks like a bug in the test. It probably needs to set AWS as the cloud provider.

Reply:
https://github.com/kubernetes/kops/pull/14517